src/chat/channel.py

```diff
@@ -199,7 +199,12 @@ def pingWindow(self):
     def openUrl(self, url):
         logger.debug("Clicked on URL: " + url.toString())
         if url.scheme() == "faflive":
-            replay(url)
+            if replay(url) == "Replays":  # game ended  -> Search Replaya
```

Comment:
I can't accept this kind of hack. If we want to pull the player id from the url, let's not hack it directly out of the url string, this is fragile as hell. I guess QUrl is the only thing we have here, but making a dedicated class for parsing and constructing these out of player / game data would be much more robust.

Reply:
when I added something to the QUrl (it was the game-uid) in the past you ripped my head of, so I worked here with what I got. 
I would have no problem with spicing up the QUrl with 1-2 more attr.
(Keep in mind, this is a fix of a pretty old bug (not a refactor), which wasn't that noticeable with the rare friend host announcement - but will be once the friend announcement will come in.)

Reply:
If I recall correctly, my issue was with the exact same thing - raw parsing of the url string. I'd really like to see a separate class for both building and parsing these game URLs.